cves/CVE-2012-4386.yml

```diff
@@ -205,4 +237,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The thing that stuck out to me about this vulnerability is that the attack
```

Comment:
So was the error that the developers made simply a code mistake, or was it a flaw in the design of this feature?

Reply:
The problem was that the code didn't verify the name of the URL's queries, even though it verified the value of the query. There wasn't really an architectural issue, the problem was that this portion of the URL wasn't considered an input, even though it's something the user can modify.